Neo4j.Driver/Neo4j.Driver/Internal/Transaction.cs

```diff
@@ -222,6 +223,10 @@ public override Task<IStatementResultCursor> RunAsync(Statement statement)
                     _connection.Server);
                 _connection.Run(statement.Text, statement.Parameters, resultBuilder);
                 await _connection.SendAsync().ConfigureAwait(false);
+
+                // Wait for SUCCESS/FAILURE message from the server
+                await _connection.ReceiveOneAsync().ConfigureAwait(false);
```

Comment:
I thought for transaction run, if we did not provide bookmark, at line 225 we would have queued messages:

```
Run Begin
Pull_All
Run transaction XX
Pull_All
```

Then if we send and just wait for the first reply, then we only got reply for `Run Begin`?

I am now wondering why tests would pass...

Reply:
I tested locally, the `receiveOneAsync` will only wait for `Success` for the first`Run Begin` message. While as we batching messages and handle them together if fits in buffer, when `Success` for `Run Begin` returns, the others messages also arrived and processed in time.